### ISSUE TITLE
Windows CRLF problem

### ISSUE THREAD

**Issue by parbo** [grailbio/bazel-compilation-database#76]
When I try to run `bazel_compdb` in my bazel workspace on Windows, I get this:

```
$ bazel_compdb
': target names may not end with carriage returns (perhaps the input source is CRLF-terminated)
': target names may not end with carriage returns (perhaps the input source is CRLF-terminated)
': target names may not end with carriage returns (perhaps the input source is CRLF-terminated)
': target names may not end with carriage returns (perhaps the input source is CRLF-terminated)
INFO: Elapsed time: 0.237s
INFO: 0 processes.
```

I can't find any target names that would end in CRLF. How can I make this work on Windows?

**Comment by parbo** [comment]
And if I run it inside bazel:

```
DEBUG: C:/b/r/zs7sfqxo/external/com_grail_bazel_compdb/aspects.bzl:341:14: Windows is not supported in compilation_database rule
```

Is Windows support planned?

**Comment by siddharthab** [comment]
I think it should be resolved with ff7a98c3e. I don't have a Windows machine to test so can not say for sure. Please reopen the issue if you think it is not working on Windows.
